src/systems/bit-media-frames.js

```diff
@@ -102,26 +114,38 @@ function isColliding(world, eidA, eidB) {
   return false;
 }
 
+// TODO we only allow capturing media-loader so rely on its bounds calculations for now
+function getEntityBounds(world, target) {
+  const targetObj = world.eid2obj.get(target);
+
+  let contentBounds;
+  if (hasComponent(world, AEntity, target)) {
+    contentBounds = targetObj.el.components["media-loader"].contentBounds;
+  } else {
+    const mediaEid = findChildWithComponent(world, MediaLoaded, target);
```

Comment:
Not a blocking issue, but a comment for future discussion.

I have been thinking that `findChild/AncestorWithComponent()` may be a bit less readable because for example it is often unclear who manages the relationships with child/ancestor entity from an entity and developers carefully need to be aware of the relationships not to break. This pattern maybe should be avoided where possible.

I added this topic to the internal design document.